.github/workflows/test.yml

```diff
@@ -18,7 +35,7 @@ jobs:
           os: [ubuntu-18.04]
     steps:
     - uses: ros-tooling/setup-ros@0.0.17
-    - uses: ros-tooling/action-ros-ci@0.0.13
+    - uses: ros-tooling/action-ros-ci@0.0.15
       with:
         package-name: libstatistics_collector metrics_statistics_msgs system_metrics_collector
```

Comment:
Do we need to remove `libstatistics_collector` from here now?

Reply:
I left it here for the code coverage report.